src/library/scala/math/Numeric.scala

```diff
@@ -134,7 +134,7 @@ object Numeric {
     def div(x: Float, y: Float): Float = x / y
   }
   trait FloatAsIfIntegral extends FloatIsConflicted with Integral[Float] {
-    def quot(x: Float, y: Float): Float = (BigDecimal(x) / BigDecimal(y)).floatValue
```

Comment:
I suggest that you add all tests in the file `test/junit/scala/math/NumericTests.scala`.
Please add a separate test for each method that you changed.